packages/mongodb-access-adapter/datagrip-access-adapter/src/main/kotlin/com/mongodb/jbplugin/accessadapter/datagrip/adapter/DataGripMongoDbDriver.kt

```diff
@@ -255,6 +218,81 @@ internal class DataGripMongoDbDriver(
             }
         }
 
+    companion object {
+        /**
+         * See https://www.mongodb.com/docs/manual/reference/mongodb-extended-json/#bson-data-types-and-associated-representations
+         * for the actual mappings.
+         */
+        @VisibleForTesting
+        internal fun deserializeEJson(doc: Any?): Any? {
+            if (doc is Map<*, *>) {
+                val mappings = doc.map { (key, value) ->
+                    when (key) {
+                        "${'$'}binary" -> null
```

Comment:
I'm not implementing mappings that we don't support in our BSON Types because we can't do anything with them. So just skip them and we can add support later whenever we need them.